src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Raw.cs

```diff
@@ -194,9 +263,37 @@ private void WriteRawValueCore(ReadOnlySpan<byte> utf8Json, bool skipInputValida
             }
 
             utf8Json.CopyTo(output.Slice(BytesPending));
-            BytesPending += len;
+            BytesPending += (int)len;
 
             SetFlagToAddListSeparatorBeforeNextItem();
         }
+
+        private void GrowIfNeed(int len)
+        {
+            // TODO (https://github.com/dotnet/runtime/issues/29293):
```

Comment:
Removed.

Reply:
Issue was closed automatically because of no activity. Unless the concern has been addressed with this PR, we should leave the comment (even if it might be pointing to a closed issue). Closed issues can be reopened, context from deleted comments is hard to recover.